src/main/resources/config.yml

```diff
@@ -48,3 +48,8 @@ layers:
     blockBlacklist:
       - LADDER
 hubSectionName: city
+guilds:
+  maxLength: 20
+  bannedWords:
+    - example
```

Comment:
make this a banned regex operation so you can specify fancier queries + we can just reuse someone else's as the default.

You probably dont need the alphanumeric checks built in that way too, could just include them here.